unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDComponentView.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class FriendsHUDComponentView : BaseComponentView, IFriendsHUDComponentView
+{
+    [SerializeField] private GameObject loadingSpinner;
+    [SerializeField] private Button closeButton;
+    [SerializeField] private Button friendsTabFocusButton;
+    [SerializeField] private Button friendRequestsTabFocusButton;
+    [SerializeField] private FriendsTabComponentView friendsTab;
+    [SerializeField] private FriendRequestsTabComponentView friendRequestsTab;
+    [SerializeField] private Model model;
+
+    public event Action<FriendRequestEntry> OnFriendRequestApproved
+    {
+        add => friendRequestsTab.OnFriendRequestApproved += value;
+        remove => friendRequestsTab.OnFriendRequestApproved -= value;
+    }
+    public event Action<FriendRequestEntry> OnCancelConfirmation
+    {
+        add => friendRequestsTab.OnCancelConfirmation += value;
+        remove => friendRequestsTab.OnCancelConfirmation -= value;
+    }
+    public event Action<FriendRequestEntry> OnRejectConfirmation
+    {
+        add => friendRequestsTab.OnRejectConfirmation += value;
+        remove => friendRequestsTab.OnRejectConfirmation -= value;
+    }
+    public event Action<string> OnFriendRequestSent
+    {
+        add => friendRequestsTab.OnFriendRequestSent += value;
+        remove => friendRequestsTab.OnFriendRequestSent -= value;
+    }
+    public event Action<FriendEntry> OnWhisper
+    {
+        add => friendsTab.OnWhisper += value;
+        remove => friendsTab.OnWhisper -= value;
+    }
+    public event Action<string> OnDeleteConfirmation
+    {
+        add => friendsTab.OnDeleteConfirmation += value;
+        remove => friendsTab.OnDeleteConfirmation -= value;
+    }
+    public event Action OnClose;
+
+    public RectTransform Transform => transform as RectTransform;
+    
+    public static FriendsHUDComponentView Create()
+    {
+        var view = Instantiate(Resources.Load<GameObject>("SocialBarV1/FriendsHUD")).GetComponent<FriendsHUDComponentView>();
+        return view;
+    }
+
+    public override void Awake()
+    {
+        base.Awake();
+        
+        friendsTabFocusButton.onClick.AddListener(() => FocusTab(0));
+        friendRequestsTabFocusButton.onClick.AddListener(() => FocusTab(1));
```

Comment:
Could we create constants for these kind of indexes?

```suggestion
        friendsTabFocusButton.onClick.AddListener(() => FocusTab(FRIENDS_TAB_INDEX));
        friendRequestsTabFocusButton.onClick.AddListener(() => FocusTab(FRIEND_REQUESTS_INDEX));
```